packages/marqus-desktop/src/main/index.ts

```diff
@@ -59,30 +60,12 @@ export async function main(): Promise<void> {
       keepAlivePromise = undefined;
     };
 
-    let quitInitiated = false;
     app.on("before-quit", async ev => {
       if (keepAlivePromise) {
         ev.preventDefault();
 
         await keepAlivePromise;
         app.quit();
-
-        return;
-      }
-
-      if (!quitInitiated) {
-        // Post-pone quitting so we can save off the log file first.
-        ev.preventDefault();
-
-        await log.close();
-
-        // Use console.log() over log.info to avoid appending this to the log file
-        // eslint-disable-next-line no-console
-        console.log(`Shutting down. Log saved to: ${log.filePath}`);
-
-        // Now let the app close.
-        quitInitiated = true;
-        app.quit();
```

Comment:
Very glad to have deleted this. It was getting confusing with that and the keepAlive promise.